WifiART/src/userinterface/ArtMvcView.java

```diff
@@ -30,13 +38,45 @@
     private static final String LCR_TAB_NAME = "Loc. Civic";
```

Comment:
Note: the abbreviations used in the GUI mock might be able to be expanded here depending on space / font-size etc.